### PR TITLE
New structure

### PR DESCRIPTION
**Pull request by r0mainK**
Related Issue: https://github.com/src-d/backlog/issues/1277

So now that changes to modelforge are all merged, minus a couple bug fixes, we can finally update this repo. In theory I could have pushed directly to the master branch with modelforge command line, but I pushed to my fork so that you could see the changes, and have a PR where to review the new templates. As you may - or not - know, now all modifications of this repo will pass through modelforge, the README and individual model files of this index now being updated simultaniously with the backend. 

@vmarkovtsev I had to changed a bit the old `nBOW` model using the `id2vec` one, since it was not compatible with the current `BOW` (tokens and documents were not included before). I also revamped the models description to represent better the current state of models.

@campoy and @marnovo (and @eiso ?), waiting for your review. Also it might be time to come to a conclusion regarding [licenses for the models](https://github.com/src-d/models/issues/3)

EDIT: also modelforge does not include DCOs as of now, not sure whether we should include it or not.

### CODE REVIEW

**Comment by vmarkovtsev** [comment]
@r0mainK We should definitely change Modelforge to add signed-off-by

**Comment by r0mainK** [comment]
By the way, don't worry about the commits, I will rebase before merge so we only have one reset commit and 4 add commits

File: bow/1e3da42a-28b6-4b33-94a2-a5671f4102f4.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| File     | https://storage.googleapis.com/models.cdn.sourced.tech/models%2Fbow%2F1e3da42a-28b6-4b33-94a2-a5671f4102f4.asdf |
+| Size     | 380.8 MB |
+| Data collection date | October 2016 |
+| Number of (sub)tokens | 999,424 |
+| Number of repositories | 112,273 |
+| License  | [](undecided) |
+
+### Dependencies
+
+* [f64bacd4-67fb-4c64-8382-399a8e7db52a]()
```
Comment (vmarkovtsev) [inline bow/1e3da42a-28b6-4b33-94a2-a5671f4102f4.md:32]: The link is empty

File: topics/c70a7514-9257-4b33-b468-27a8588d4dfa.md [hunk 2]
```diff
@@ -18,13 +19,14 @@ print("Number of topics:", len(topics))
...
 | Data collection date | October 2016 |
-| Number of (sub)tokens | 2 million |
+| Number of tokens | 2,015,336 |
+| Number of topics | 320 |
 | License  | [](undecided) |
 
 ### Dependencies
 
-* [f64bacd4-67fb-4c64-8382-399a8e7db52a](/docfreq/f64bacd4-67fb-4c64-8382-399a8e7db52a.md)
+* [f64bacd4-67fb-4c64-8382-399a8e7db52a]()
```
Comment (vmarkovtsev) [inline topics/c70a7514-9257-4b33-b468-27a8588d4dfa.md:32]: Same here

**Review by vmarkovtsev — APPROVED** [review]

**Comment by vmarkovtsev** [comment]
@r0mainK We need to fix the links to dependencies.

File: README.md [hunk 3]
```diff
@@ -1,62 +1,64 @@
...
+=========================
 
-## docfreq
-Document frequencies of source code identifiers, that is, how many projects contain an identifier.
+## bow
+Weighted bag-of-words, that is, every bag is a feature extracted from source code and associated with a weight obtained by applying TFIDF.
 
 Example:
 
 ```
```
Comment (campoy) [inline README.md:9]: "```python" so it shows as python code

File: README.md [hunk 4]
```diff
@@ -1,62 +1,64 @@
...
+from sourced.ml.models import BOW
+bow = BOW().load(bow)
+print("Number of documents:", len(bow))
+print("Number of tokens:", len(bow.tokens))
 ```
 
 1 model:
 
-* <default> [f64bacd4-67fb-4c64-8382-399a8e7db52a](/docfreq/f64bacd4-67fb-4c64-8382-399a8e7db52a.md) 
+* <default> [1e3da42a-28b6-4b33-94a2-a5671f4102f4](/bow/1e3da42a-28b6-4b33-94a2-a5671f4102f4.md)
```
Comment (campoy) [inline README.md:18]: what are these for?
there's no documentation explaining what these numbers mean

File: README.md (hunk 3 above)
Comment (vmarkovtsev) [reply README.md:9]: :+1:

File: README.md (hunk 4 above)
Comment (vmarkovtsev) [reply README.md:18]: Each model has a UUID. Also each type has a default UUID. Indeed, we should document this.
Roughly the same way docker images work, "latest" is "default" here, and sha256s are UUIDs.

File: README.md (hunk 3 above)
Comment (r0mainK) [reply README.md:9]: done: https://github.com/src-d/modelforge/pull/43

File: README.md (hunk 4 above)
Comment (r0mainK) [reply README.md:18]: done: https://github.com/src-d/modelforge/pull/44

**Pull request merged**